src/resolvers/index.test.ts

```diff
@@ -83,17 +66,13 @@ describe('GraphQL resolvers', () => {
 
   describe('without being logged in', () => {
     beforeAll(async () => {
-      server = new ApolloServer({
+      server = new ApolloServer<ServerContext>({
```

Comment:
we don't need to worry about passing in the context here, since this is testing for a logged out user